data/multiplayer/eras.lua

```diff
@@ -97,6 +97,8 @@ res.turns_over_advantage = function()
 	-- if #winning_sides==0, then every side either has no units or has a negative score
 
 	-- po: "Turns Over", meaning "turn limit reached" is the title of the end-of-match summary dialog
-	local a, b = gui.show_popup(_ "dialog^Turns Over", side_comparison)
+	title = title or _ "dialog^Turns Over"
+	local a, b = gui.show_popup(title, side_comparison)
```

Comment:
Is there any point in having `local a, b` here? It's my fault that they were added, but I can't remember why I added them.

Reply:
I'm really not sure why it's there. `gui.show_popup` doesn't return anything, though, so it seems both variables will always be `nil` anyway.